### DISCUSSION TITLE
I can't  open  file ending in xxx.sh

### DISCUSSION THREAD

**Discussion by S-loan** [Eugeny/tabby#6275] [Q&A]
hi , After i downLoad version 1.7.0 , I can't open file  ending in .sh. 
For example, file name is xxl.sh.   Double click later Nothing happened.But, I manual input command,This will run successfully.
I hope you can help me solve this problem. Thank you🙏🏻

**Comment by Eugeny** [comment]
I think you've accidentally posted to a wrong repo

**Reply by S-loan** [reply]
No, man. I really have this problem.I use the default terminal of macOS to execute . sh files.I'm not sure what the problem is....😭

**Reply by S-loan** [reply]
I execute a line of command, for example  java- jar ......
This type is executable on the Tabby console. But I'm going to put this in the.sh file,Double click later Nothing happened😭
I don't know what the problem is

**Reply by Eugeny** [reply]
Tabby has nothing to do with double clicking files in the OS. For opening a .sh file to work, it only needs to have executable perms and a hashbang line in front.

**Reply by S-loan** [reply]
Ok, I see. Thank you😊
